tailwind.config.cjs

```diff
@@ -1,12 +1,19 @@
 module.exports = {
+  darkMode: "selector",
   content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
   theme: {
     extend: {
       fontFamily: {
         satoshi: ["Satoshi", "sans-serif"],
+        inter: ["Inter", "sans-serif"],
       },
       colors: {
-        "off-white": "#F2F2F2",
+        "off-white": "#F3F3F3",
+        "light-blue": "#D0E1EB",
+        "light-gray": "#6F6F6F",
+        "medium-gray": "#454545",
+        "dark-gray": "#555555",
+        "luna-purple": "#C4BBFB",
```

Comment:
Ideally the colors would be in the TW format, makes it easier to add more colors or change colors later on. 
```suggestion
        "off-white": "#F3F3F3",
        "blue": {
             100: "#D0E1EB"
        },
        gray: {
            100: "#6F6F6F",
            400: "#454545",
            800: "#555555",
        }
        "luna-purple": "#C4BBFB",
```